e2e-cypress/files/support/commands.ts

```diff
@@ -7,6 +7,127 @@
 // https://on.cypress.io/custom-commands
 // ***********************************************
 
+// Azure SSO with MSALv2
+// ***********************************************
+// Originally developed by @tbabik and @herrkoch
+//
+// ATTENTION: This approach requires a technical, so-called "cloud-only" Azure user (<username>@<domain>.onmicrosoft.com) which does not
+// go through federation. (For Boehringer Ingelheim coders: Please refer to the "BI Coders Notebook", section "Cypress and Azure SSO" in
```

Comment:
I would avoid any BI-specific remarks within ODS. Maybe write it like this:
"For security reasons those kind of users might require special adjustments within your organization. Please refer to your company's documentation around this topic or ask the IT department"

Reply:
Agreed, we moved all of the BI-specific information out of the quickstarter but I wanted to leave this little hint in there so BI folks know where to look. :-) @michaelsauter said he would be fine with that. ;-)